spec/controllers/article_controller_spec.rb

```diff
@@ -7,4 +7,27 @@
       expect(response).to render_template('new')
     end
   end
+
+  it 'handles authentication'
+  it 'handles configuration'
```

Comment:
Are these `TODO` tests?  (I also see a few others in other places, just not familiar w/ this pattern)